torch/_dynamo/codegen.py

```diff
@@ -50,7 +50,7 @@ def __init__(
         tx=None,
         root: Optional[torch.nn.Module] = None,
         graph_output_var: Optional[str] = None,
-        tempvars=None,
+        tempvars: Dict[VariableTracker, str] = None,
```

Comment:
```suggestion
        tempvars: Optional[Dict[VariableTracker, str]] = None,
```

Reply:
I ended up getting rid of this because the `str` is also an `Optional[str]`, but in `SideEffects` we also assume it'll be a `str` at places.